src/components/InfiniteScrollList.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { GridList } from '@material-ui/core'
+import InfiniteScroll from 'react-infinite-scroller'
+import styled from 'styled-components'
+import ListLoader from '~Components/ListLoader'
+
+interface InfiniteScrollListProps {
+  children: React.ReactNode
+  hasMoreCharactersToLoad: boolean
+  loadMoreCharacters: () => void
```

Comment:
I think that this should be generic too 👍

Reply:
Yes, totally. I will change it